Lib/unittest/suite.py

```diff
@@ -231,21 +232,22 @@ def _handleModuleFixture(self, test, result):
                 _call_if_exists(result, '_restoreStdout')
 
     def _createClassOrModuleLevelException(self, result, exc, method_name,
-                                           parent, info=None):
+                                           parent, err=None):
         errorName = f'{method_name} ({parent})'
-        self._addClassOrModuleLevelException(result, exc, errorName, info)
+        self._addClassOrModuleLevelException(result, exc, errorName, err)
 
     def _addClassOrModuleLevelException(self, result, exception, errorName,
-                                        info=None):
+                                        err=None):
         error = _ErrorHolder(errorName)
         addSkip = getattr(result, 'addSkip', None)
         if addSkip is not None and isinstance(exception, case.SkipTest):
             addSkip(error, str(exception))
         else:
-            if not info:
-                result.addError(error, sys.exc_info())
+            if not err:
```

Comment:
It is not clear why the assert it needed, but with it, a possible refactoring (which cannot be marked as a multi-line suggestion).
```
            if not err:
                err = sys.exc_info()[1]
            else:
                assert not isinstance(err, tuple)
            result.addError(error, err)
```